prime-router/src/main/kotlin/history/azure/HistoryDatabaseAccess.kt

```diff
@@ -148,6 +229,20 @@ abstract class HistoryDatabaseAccess(
     ): Condition {
         var filter = this.organizationFilter(organization, orgService)
 
+        if (receivingOrgSvcStatus != null) {
+            filter = if (receivingOrgSvcStatus.toString() == CustomerStatus.ACTIVE.toString()) {
```

Comment:
I think you can drop the `toString` here

Reply:
Actually, can this not all be simplified?

```
filter.and(
                    DSL.jsonbGetAttributeAsText(SETTING.VALUES, "customerStatus")
                    .notEqual(customerStatus.name)
                )
```
?

Reply:
still need the lowercase() since the customerStatus is stored in the dbs as lowercase.

Reply:
If you use `name` instead of `toString` I think it will work?